core/pom.xml

```diff
@@ -25,6 +25,21 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-core</artifactId>
+      <version>${metrics.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-jvm</artifactId>
+      <version>${metrics.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-graphite</artifactId>
```

Comment:
Is the Graphite dependency required?

At least some versions of Graphite are/were LGPL licensed which will create licensing issues for downstream Tachyon consumers such as Apache Spark where LGPL dependencies may only be optional

It looks like the version used is probably ASLv2 which is not problematic but would be useful if you could confirm this

If the graphite dependency is not used (I can't see that it is) can we just remove it for now